ckanext/unhcr/templates/package/read_base.html

```diff
@@ -53,6 +53,8 @@
       {% set dataset_extent = h.get_geojson_geometries(pkg.geographies) %}
       {% if dataset_extent %}
         {% snippet "spatial/snippets/dataset_map_sidebar.html", extent=dataset_extent %}
+      {% else %}
+      <p>No map available for the selected dataset geography</p>
```

Comment:
This will be shown to datasets that don't have any geographies assigned right? If so let's say
```suggestion
      <p>No geographic coverage defined</p>
```

Reply:
No @amercader, it also include defined geographies that don't have a geometry defined. 
I'm proposing [this change](https://github.com/okfn/ckanext-unhcr/commit/faaaf830fe1cabc5c5e9f1ea9062f05521bedd0c)

```html
      {% set dataset_extent = h.get_geojson_geometries(pkg.geographies) %}
      {% if dataset_extent %}
        {% snippet "spatial/snippets/dataset_map_sidebar.html", extent=dataset_extent %}
      {% elif pkg.geographies %}
      <p>No map available for the selected dataset geography</p>
      {% else %}
      <p>No geographic coverage defined</p>
      {% endif %}
```